006-strimzi-ui.md

```diff
@@ -114,6 +114,28 @@ Finally, one capability which may be of interest (which may have bearing on how
 
 In all of these cases, capabilities can be added in a prioritised order, and should be added in a phased manner themselves (for example, add a view of the brokers and their configuration, then the ability to modify select configuration).
 
+#### Proposed deployment
+
+This UI could be deployed as a part of Strimzi as follows:
+
+![Suggested deployment](./images/006-deployment.png)
+
+Where:
+
+- The UI is deployed standalone, akin to how the Kafka bridge, Kafka Connect are currently for example (so a new CRD would be required, with configuration needed to reference which cluster(s) to interact with).
+- The UI's spec would contain 0 to N Kafka clusters, which: 
+  - Contain 1 or more 'backend' entries. These represent backend sources of data which can be surfaced in the UI.
+  - One of these entries will be the `strimzi-api` (ie each Kafka kind will contain its own api server).
+  - Each of these entries will define items such as the address to use to connect to it, any auth credentials required, Transport security required, etc.
+- The UI's spec will contain general configuration for the UI - such as certificates to expose to clients etc.
```

Comment:
@matthew-chirgwin can you provide an example of these things configured in the new Web UI resource you are mentioning and if needed the additions in the Kafka one, please?

Reply:
Hi @ppatierno - sorry for the delay in replying. Of course - I will come up with some pseudo-yaml to represent the above in context and add it to the proposal.

Reply:
@ppatierno I have just pushed a new commit with some more details - please let me know if there is more you would like me to elaborate on